plum/util.py

```diff
@@ -122,7 +122,7 @@ def is_in_class(f: Callable) -> bool:
 
 
 def _split_parts(f: Callable) -> List[str]:
-    qualified_name = f.__module__ + "." + f.__qualname__
+    qualified_name = (f.__module__ or "") + "." + f.__qualname__
```

Comment:
Shouldn't `f` always have a `__module__`? Or is it during doctests that `__module__` may not be set?

If `__module__` is not set, would it be more correct to just return `f.__qualname__.split(".")`?

Reply:
Agreed!

Reply:
I used a ternary